modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3228,15 +3228,82 @@ curl --fail -u <my_username>:<my_password> -X POST "http://localhost:14240/gsqls
  }'
 ----
 
-=== List running queries
+=== List all running queries
+`GET /showprocesslistall`
 
-`+GET /showprocesslist/{graph_name}+`
+This endpoint reports the statistics of all running queries across all graphs on a TigerGraph instance: the query's request ID, start time, expiration time, and the REST endpoint's URL.
+This includes installed queries. interpreted queries, as well as built-in queries.
 
-This endpoint reports statistics of running queries of a graph: the query's request ID, start time, expiration time, and the REST endpoint's URL.
+If you are running a TigerGraph cluster, this endpoint only shows queries running on the node to which the request is sent, not all running queries on the cluster.
+
+This endpoint requires the `READ_DATA` privilege on the global scope.
+
+==== Sample request
+[tabs]
+====
+Request::
++
+[.wrap,console]
+----
+$ curl -X GET "http://localhost:9000/showprocesslistall" |jq
+----
+Response::
++
+----
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100   671  100   671    0     0   655k      0 --:--:-- --:--:-- --:--:--  655k
+{
+  "version": {
+    "edition": "enterprise",
+    "api": "v2",
+    "schema": 0
+  },
+  "error": false,
+  "message": "",
+  "results": [
+    {
+      "requestid": "15.RESTPP_1_1.1660165648633.N", <1>
+      "startTime": "2022-08-10 21:07:28.633",
+      "expirationTime": "2022-08-10 21:07:44.633", <2>
+      "url": "/query/BrandNewSocial/rngExample",
+      "elapsedTime": 3385
+    },
+    {
+      "requestid": "65551.RESTPP_1_1.1660165648638.N",
+      "startTime": "2022-08-10 21:07:28.638",
+      "expirationTime": "2022-08-10 21:07:44.638",
+      "url": "/query/Social/rngExample",
+      "elapsedTime": 3380
+    },
+    {
+      "requestid": "131087.RESTPP_1_1.1660165648723.N",
+      "startTime": "2022-08-10 21:07:28.723",
+      "expirationTime": "2022-08-10 21:07:44.723",
+      "url": "/query/NewSocial/rngExample",
+      "elapsedTime": 3295
+    }
+  ],
+  "code": "REST-0000"
+}
+----
+<1> Each request to run a query sent to the REST server has a unique ID that can be used to look up the request in the system logs.
+<2> The time at which the query will time out.
+====
 
+==== Parameters
+No parameters.
+
+=== List running queries of a graph
+
+`GET /showprocesslist/{graph_name}`
```

Comment:
can you move this API ahead of showprocesslistall API (so they reflect the timeline on when they were implemented and as people prefer to use in the order of appearance, this /showprocesslist/<graph> is less intrusive than the new one).